addons/l10n_fr/models/account_chart_template.py

```diff
@@ -7,6 +7,32 @@
 class AccountChartTemplate(models.Model):
     _inherit = 'account.chart.template'
 
+    l10n_fr_rounding_difference_loss_account_id = fields.Many2one(
+        'account.account.template',
+        string="Loss Account for Rounding Differences",
+        help="Account used for losses from rounding the lines of French tax returns",
+    )
+    l10n_fr_rounding_difference_profit_account_id = fields.Many2one(
+        'account.account.template',
+        string="Profit Account for Rounding Differences",
+        help="Account used for profits from rounding the lines of French tax returns",
+    )
+
+    def _load_company_accounts(self, account_ref, company):
+        super()._load_company_accounts(account_ref, company)
+
+        # Add the fields that will contain the accounts used for the
+        # profit/loss from rounding lines of the french tax report
+        if company.country_id.code == 'FR':
```

Comment:
Why only if the company's country code is `== 'FR'`?
If a company in another country wants to load the chart template, so be it :shrug: 
For instance, this is a country code I would expect uses the same chart template and roundings https://www.iso.org/obp/ui/#iso:code:3166:MQ